ethclient/ethclient.go

```diff
@@ -662,6 +662,16 @@ func toCallArg(msg ethereum.CallMsg) interface{} {
 	if msg.GasTipCap != nil {
 		arg["maxPriorityFeePerGas"] = (*hexutil.Big)(msg.GasTipCap)
 	}
+	if msg.AccessList != nil {
+		accessListArg := make([]map[string]interface{}, len(msg.AccessList))
```

Comment:
The conversion is not necessary because `types.AccessList` already has the correct JSON encoding. So we can just put the access list into the args and done.

Reply:
Yeah, I tested locally and this change works, too. btw, the CI seems to fail with:
```
--- FAIL: TestForkResendTx (0.03s)
    backend_test.go:234: sending transaction: nonce too low: next nonce 1, tx nonce 0
FAIL
FAIL	github.com/ethereum/go-ethereum/ethclient/simulated	0.157s
```